docs/reference/search/search-your-data/search-using-query-rules.asciidoc

```diff
@@ -0,0 +1,150 @@
+[[search-using-query-rules]]
+== Searching with query rules
+++++
+<titleabbrev>Searching with query rules</titleabbrev>
+++++
+
+[[query-rules]]
+
+preview::[]
+
+_Query rules_ are rules that are defined inside {es} that allow customization of search results for queries that match specified criteria metadata.
+This allows for more control over promoted results, ensuring that specified documents always match defined criteria and are returned at the top of the result list.
```

Comment:
I think we should keep open the goal of query rules. Maybe we could move this to the pinned query rule type, or add a paragraph explaining that right now this is the only use for query rules...

Reply:
I moved this here based on earlier feedback, to make it less confusing that _right now_ query rules are for pinned documents only. I added a small clarification that should help, though, WDYT?